### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by JC4github**
- All pages are now grouped in a pages folder.
- Created a new assets folder in the project root, the SVG assets are now extracted into their own files from the index page. 
- Changed the name of the tab to UoA Degree Planner.

### CODE REVIEW

**Review by jmadelino — APPROVED** [review]
LGTM! I am sure as we start delving deeper into the code we will find other thinks to refactor, but as a high level refactor this looks great!

File: app/pages/courses/page.tsx
Comment (jaskfla) [inline app/pages/courses/page.tsx:1]: How come the diff things every line in this file got changed? 🤨

**Review by jaskfla — COMMENTED** [review]
Looks like a nice improvement! I’m just a bit baffled why the Git diff seems to think one of the files got every line changed. It doesn’t look like you altered it at all?

Did something get auto-formatted? I couldn’t spot anything.

File: app/pages/courses/page.tsx
Comment (JC4github) [reply app/pages/courses/page.tsx:1]: no idea but I just moved the folder into the pages folder like all the other ones

**Pull request merged**